src/components/Header/index.tsx

```diff
@@ -1,16 +1,29 @@
 'use client';
 
 import * as S from './style';
+import * as C from '../index';
+import * as I from '@/assets';
+import React, { useState } from 'react';
 
-import { LogoIcon } from '@/assets';
-
-const Header = () => (
-  <S.Header>
-    <S.Inner>
-      <LogoIcon />
-      <S.PageName>취업자 리스트 조회</S.PageName>
-    </S.Inner>
-  </S.Header>
-);
+const Header = () => {
+  const [isDrop, setIsDrop] = useState<boolean>(false);
+  return (
+    <>
+      <S.Header>
+        <S.Inner>
+          <I.LogoIcon />
+          <S.IconContainer
+            onClick={(e: React.MouseEvent<HTMLDivElement>) => {
+              setIsDrop((prev) => !prev);
```

Comment:
```suggestion
              setIsDrop(!isDrop);
```

람다를 사용하지 않는것이 더 직관적으로 보입니다!

Reply:
함수형 업데이트를 통해 state의 최신 값을 보장받을 수 있습니다.

prev라는 변수 명이 아쉽다면 아래처럼 바꿔도 좋다고 생각합니다.

```tsx
setIsDrop((isDrop) => !isDrop)
```

Reply:
넵 좋은것 같습니다!